Godeps/_workspace/src/github.com/jbenet/go-temp-err-catcher/README.md

```diff
@@ -0,0 +1,78 @@
+# go-temp-err-catcher
+
+This is a little package to use with your net.Listeners.
+
+Docs: https://godoc.org/github.com/jbenet/go-temp-err-catcher
+
+Get:
+
+  go get github.com/jbenet/go-temp-err-catcher
+
+## Examples
+
+It is meant to be used with things like net.Lister.Accept:
+
+```go
+import (
+  tec "github.com/jbenet/go-temp-err-catcher"
+)
+
+func listen(listener net.Listener) {
+  var c tec.TempErrCatcher
+
+  for {
+    conn, err := listener.Accept()
+    if err != nil && c.IsTemporary(c) {
```

Comment:
I feel like were throwing errors away here that we may want to at least note somehow... maybe it should also take a `func(err error)` so we can log those errors if we want?